pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/stateful/ReadingState.java

```diff
@@ -0,0 +1,63 @@
+package io.pravega.sensor.collector.stateful;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+public class ReadingState {
```

Comment:
Add a JavaDoc comment explaining the purpose of this class.

Reply:
Done